src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -234,22 +229,10 @@ public static NodeSettings Default(Network network = null, ProtocolVersion proto
             nodeSettings.SyncTimeEnabled = config.GetOrDefault<bool>("synctime", true);
             nodeSettings.Logger.LogDebug("Time synchronization with peers is {0}.", nodeSettings.SyncTimeEnabled ? "enabled" : "disabled");
 
-            nodeSettings.UseCheckpoints = config.GetOrDefault<bool>("checkpoints", true);
-            nodeSettings.Logger.LogDebug("Checkpoints are {0}.", nodeSettings.UseCheckpoints ? "enabled" : "disabled");
-
-            // TODO: Figure out where to put defaults so they are easy to configure.
-            uint256 defaultAssumeValid = null;
-            if (!nodeSettings.Network.IsBitcoin() && nodeSettings.Network.IsTest())
-            {
-                // Block Height 184096 https://chainz.cryptoid.info/strat-test/block.dws?184096.htm
-                defaultAssumeValid = new uint256("0x74427b2f85b5d9658ee81f7e73526441311122f2b23702b794be557ba43ca43e");
-            }
-            // TODO: Add defaults for bitcoin and mainnet networks.
-
-            nodeSettings.BlockAssumedValid = config.GetOrDefault<uint256>("assumevalid", defaultAssumeValid);
-            if (nodeSettings.BlockAssumedValid == 0) // 0 means validate all blocks
-                nodeSettings.BlockAssumedValid = null;
-
+            nodeSettings.Consensus.Load(config, nodeSettings.Network);
```

Comment:
Hi mike one more comment, ConsensusSettings can be standalone and not inside the Settings file (see other examples like StoreSettings) that way we don't bind consensus setting to the base feature.